personalwebsite/projects/templates/projects/projects.html

```diff
@@ -27,12 +29,8 @@ <h1 class="display-5 font-family-roboto">Projects</h1>
                 <h5>{{ project.title }}</h5>
                 {{ project.description }}
             </div>
-            <div class="tags d-flex flex-wrap mt-1">
-                {% if project.tags.all %}
-                {% for tag in project.tags.all %}
-                <div class="tag-label">{{ tag }}</div>
-                {% endfor %}
-                {% endif %}
+            <div class="tags">
+                {% show_tags project %}
```

Comment:
Good work incorporating inclusion tags like this here. I like how reusable this will be moving forward.